entrypoint.sh

```diff
@@ -207,11 +215,11 @@ function seiso_lint() {
 	included=()
 
 	while read -r file; do
-		if [[ -n ${INPUT_EXCLUDE:+x} && "${file}" =~ ${INPUT_EXCLUDE} ]]; then
+		if [[ -n ${FILTER_REGEX_EXCLUDE:+x} && "${file}" =~ ${FILTER_REGEX_EXCLUDE} ]]; then
```

Comment:
We were mapping these variables earlier in the script, so I changed them here for consistency with other vars.